site-src/index.md

```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+## What is the Multicluster API?
```

Comment:
The content here is generally more specific to the MCS API. The word "Multicluster API" is throwing me off lol. I think this page should be more general about "SIG-Multicluster" and what the SIG works on (like our charter, the projects and problems we talk about in the maintainer track talks). It so happens that some of the content here still generally applies to all problems SIG-MC is in charter for so we can work together on how to chop this up as such.